diff --git a/src/win32/cairo-dwrite-font.cpp b/src/win32/cairo-dwrite-font.cpp
index 0a3b773..7c9a4df 100644
--- a/src/win32/cairo-dwrite-font.cpp
+++ b/src/win32/cairo-dwrite-font.cpp
@@ -740,15 +740,13 @@ public:
 
     cairo_fixed_t GetFixedX(const D2D1_POINT_2F &point)
     {
-	unsigned int control_word;
-	_controlfp_s(&control_word, _CW_DEFAULT, MCW_PC);
+	_controlfp(_CW_DEFAULT, MCW_PC);
 	return _cairo_fixed_from_double(point.x);
     }
 
     cairo_fixed_t GetFixedY(const D2D1_POINT_2F &point)
     {
-	unsigned int control_word;
-	_controlfp_s(&control_word, _CW_DEFAULT, MCW_PC);
+	_controlfp(_CW_DEFAULT, MCW_PC);
 	return _cairo_fixed_from_double(point.y);
     }
 
